lib/aion_cli/commands/data.rb

```diff
@@ -169,7 +170,7 @@ def stats_3f(path)
           area_voters = rows.select { |row| row[index_area_number] == area_number }
           area_label = "#{area_number} #{area_text}"
 
-          eligible_voters = area_voters.size
+          eligible_voters = total_json[headers[index_area_number]][area_number] || 0
```

Comment:
Dig incase of missing keys?
```suggestion
          eligible_voters = total_json.dig(headers[index_area_number], area_number) || 0
```